packages/dart/noports_core/lib/src/commands/utils/usage_messages.dart

```diff
@@ -0,0 +1,74 @@
+import 'package:chalkdart/chalk.dart';
+import 'package:noports_core/src/commands/issue_keys/issue_keys_params.dart';
+
+class UsageMessages {
+  static final mainMenu =
+      '''
+  
+${chalk.bold('Usage:')} noports ${chalk.cyan('<command>')} [options]
+
+${chalk.bold('Commands:')}
+  ${chalk.cyan('activate')}      Activate your atsign (new user onboarding)
+  ${chalk.cyan('issue-keys')}    Create a new enrollment
+ 
+${chalk.bold('Examples:')}
+  ${chalk.cyan('activate')}      noports activate '@alice:enroll:otp:123456' --keyfile ~/.mykeys.json
+  ${chalk.cyan('issue-keys')}    noports issue-keys -a @alice -k /home/user/keys/@alice.atKeys
+  
+Run 'noports ${chalk.cyan('<command>')} --help' for more information
+''';
+
+  static final activateHelp =
+      '''
+${chalk.bold('Usage:')} noports activate ${chalk.cyan('<activation_string>')} [options]
+
+${chalk.bold('Options:')}
+  -k, --keyfile <path>        Path to save atKeys file
```

Comment:
Looks like the help wasn't updated

Reply:
I did update it earlier, I also reworked them again through ([8a4a9d7](https://github.com/atsign-foundation/noports/pull/2304/commits/8a4a9d7ed63ad7593a82669e699a1dbda2b53f6f)).